package.json

```diff
@@ -15,7 +17,17 @@
   "author": "GitHub",
   "license": "CC0-1.0",
   "devDependencies": {
+    "mocha": "^5.2.0",
+    "spectron": "^5.0.0"
+  },
+  "dependencies": {
     "electron": "^4.0.1",
-    "electron-tooltip": "^1.1.5"
+    "electron-tooltip": "^1.1.5",
+    "@google-cloud/speech": "^2.2.0",
+    "@google-cloud/storage": "^2.4.1",
+    "jquery": "^3.3.1",
+    "read-blob": "^1.1.0",
+    "recorder-js": "^1.0.7",
+    "web-audio-beat-detector": "^7.0.26"
```

Comment:
We can remove this dependency I don't think we use it anymore?

Reply:
Aye